composites/Plugin/Shared/components/IconButtonToggle.js

```diff
@@ -21,10 +21,24 @@ const ChangingIconButtonBase = styled.button`
 	height: ${ props => props.pressed ? "23px" : "24px" };
 
 	&:hover {
-		border-color: ${ props => props.hoverBorderColor }
+		border-color: ${ props => props.hoverBorderColor };
+	}
+	&:disabled {
+		background-color: ${ props => props.unpressedBackground };
+		box-shadow: none;
+		border: none;
+		cursor: default;
 	}
 `;
 
+let areButtonsDisabled = function( props ) {
```

Comment:
You might consider returning a default bool. Because now this method returns two different types: `void` and `bool`